MIP113/MIP113.md

```diff
@@ -443,6 +443,7 @@ List of active AVCs and their members:
 | AVC Member Name | AVC Member Address | Forum Verification | Verified Signature | Verified MKR Holding (MKR) (3dp) | Approved to Join AVC |
 |---|---|---|---|---:|---|
 | seedlatam.eth | [0x0087a081a9b430fd8f688c6ac5dd24421bfb060d](https://etherscan.io/address/0x0087a081a9b430fd8f688c6ac5dd24421bfb060d) | [Link](https://forum.makerdao.com/t/cvc-member-recognition-submission-seedlatam-eth/20836) | [Link](https://etherscan.io/verifySig/18891) | 6.800 | [AVC Creator](https://forum.makerdao.com/t/cvc-creation-sovereign-finance-cvc/20868/) |
+| 0xRoot | [0xC74392777443a11Dc26Ce8A3D934370514F38A91](https://etherscan.io/address/0xC74392777443a11Dc26Ce8A3D934370514F38A91) | [Link](https://forum.makerdao.com/t/avc-member-recognition-0xroot/22267) | [Link](https://etherscan.io/verifySig/25450) | 5.01959484 | [AVC Creator](https://forum.makerdao.com/t/cvc-creation-sovereign-finance-cvc/20868) |
```

Comment:
@blimpa The "Approved to Join AVC" link should be a link pointing to the post in which the AVC is approving the member to join the AVC, in this case, it's https://forum.makerdao.com/t/avc-member-recognition-0xroot/22267/2